ElasticBox/user-authentication.md

```diff
@@ -89,7 +145,15 @@ As good practice, you should give an LDAP user in your organization administrati
 3. Make the LDAP user an administrator. From here on, use that LDAP user account to manage ElasticBox.
 
 ### Contacting ElasticBox Support
+<<<<<<< HEAD
```

Comment:
Clean up this section. remove the `<<<<<<< HEAD`, `=======`, and `>>>>>>> CenturyLinkCloud/master` lines. Also remove the duplicate content.